airbyte-integrations/connectors/source-linkedin-ads/source_linkedin_ads/utils.py

```diff
@@ -283,7 +297,8 @@ def transform_variables(
         record["variables"]["type"] = key
         record["variables"]["values"] = []
         for key, param in params.items():
-            record["variables"]["values"].append({"key": key, "value": param})
+            value = str(param)  # convert various datatypes of values into the string
+            record["variables"]["values"].append({"key": key, "value": value})
```

Comment:
why conversation to string? it would be better to use JSON format if this data is parsed another service/connector with not Python software.

Reply:
Json is fine, corrected.